### ISSUE TITLE
Errors while installing T3 packages

### ISSUE THREAD

**Issue by max-programming** [nachoaldamav/ultra#141]
### Provide installed version

`[Ultra] v0.7.6`

### Provide Node.js version

16.17.1

### Current platform

Windows

### Describe the Bug

I got these weird error logs while trying to install with `ultra i` after running `ultra create t3-app`

```
ℹ Using https://registry.npmjs.org/ as registry...
⚡ Installed local packages in 0.15 ms!
✖ Error while installing globals@^13.15.0 from cache - Error reading package.json from C:\Users\Admin\.ultra-cache\globals\13.17.0\package.json - ENOENT: no such file or directory, open 'C:\Users\Admin\.ultra-cache\globals\13.17.0\package.json'
✖ Error extracting C:\Users\Admin\AppData\Local\Temp\ultra_tmp\ignore-b50a0b7a023c81aa1a217fb5dca7ff67.tgz to C:\Users\Admin\.ultra-cache\ignore\5.2.0: TAR_BAD_ARCHIVE: Unrecognized archive format
✖ Error while installing @eslint/eslintrc@^1.3.0 from cache - Error reading package.json from C:\Users\Admin\.ultra-cache\ignore\5.2.0\package.json - ENOENT: no such file or directory, open 'C:\Users\Admin\.ultra-cache\ignore\5.2.0\package.json'
✖ Error extracting C:\Users\Admin\AppData\Local\Temp\ultra_tmp\path-exists-7c15fc7e9a3b16862093ccba6ad29a03.tgz to C:\Users\Admin\.ultra-cache\path-exists\4.0.0: TAR_BAD_ARCHIVE: Unrecognized archive format
✖ Error while installing find-up@^5.0.0 from cache - Error reading package.json from C:\Users\Admin\.ultra-cache\path-exists\4.0.0\package.json - ENOENT: no such file or directory, open 'C:\Users\Admin\.ultra-cache\path-exists\4.0.0\package.json'
✖ Error while installing globals@^13.15.0 from cache - Error reading package.json from C:\Users\Admin\.ultra-cache\globals\13.17.0\package.json - ENOENT: no such file or directory, open 'C:\Users\Admin\.ultra-cache\globals\13.17.0\package.json'
✖ Error extracting C:\Users\Admin\AppData\Local\Temp\ultra_tmp\read-cache-7a65d6fcfaf1ca051c995b784c6eb1f7.tgz to C:\Users\Admin\.ultra-cache\read-cache\1.0.0: TAR_BAD_ARCHIVE: Unrecognized archive format
✖ Error while installing postcss-import@^14.1.0 from cache - Error reading package.json from C:\Users\Admin\.ultra-cache\read-cache\1.0.0\package.json - ENOENT: no such file or directory, open 'C:\Users\Admin\.ultra-cache\read-cache\1.0.0\package.json'
✖ Error while installing ignore@^5.2.0 from cache - Error reading package.json from C:\Users\Admin\.ultra-cache\ignore\5.2.0\package.json - ENOENT: no such file or directory, open 'C:\Users\Admin\.ultra-cache\ignore\5.2.0\package.json'
✖ Error while installing ignore@^5.2.0 from cache - Error reading package.json from C:\Users\Admin\.ultra-cache\ignore\5.2.0\package.json - ENOENT: no such file or directory, open 'C:\Users\Admin\.ultra-cache\ignore\5.2.0\package.json'
✖ Error extracting C:\Users\Admin\AppData\Local\Temp\ultra_tmp\levn-5f3062071eebdde1684e0090ae90e913.tgz to C:\Users\Admin\.ultra-cache\levn\0.4.1: TAR_BAD_ARCHIVE: Unrecognized archive format
✖ Error while installing eslint@8.22.0 from cache - Error reading package.json from C:\Users\Admin\.ultra-cache\levn\0.4.1\package.json - ENOENT: no such file or directory, open 'C:\Users\Admin\.ultra-cache\levn\0.4.1\package.json'
✖ Error while installing ignore@^5.2.0 from cache - Error reading package.json from C:\Users\Admin\.ultra-cache\ignore\5.2.0\package.json - ENOENT: no such file or directory, open 'C:\Users\Admin\.ultra-cache\ignore\5.2.0\package.json'
✖ Error extracting C:\Users\Admin\AppData\Local\Temp\ultra_tmp\regexpp-850ab2db51deef81dcd32b6851513535.tgz to C:\Users\Admin\.ultra-cache\regexpp\3.2.0: TAR_BAD_ARCHIVE: Unrecognized archive format
✖ [ERR] @typescript-eslint/eslint-plugin@^5.33.0 - Error reading package.json from C:\Users\Admin\.ultra-cache\regexpp\3.2.0\package.json - ENOENT: no such file or directory, open 'C:\Users\Admin\.ultra-cache\regexpp\3.2.0\package.json'
✖ Error while installing globals@^13.15.0 from cache - Error reading package.json from C:\Users\Admin\.ultra-cache\globals\13.17.0\package.json - ENOENT: no such file or directory, open 'C:\Users\Admin\.ultra-cache\globals\13.17.0\package.json'
✖ Error while installing ignore@^5.2.0 from cache - Error reading package.json from C:\Users\Admin\.ultra-cache\ignore\5.2.0\package.json - ENOENT: no such file or directory, open 'C:\Users\Admin\.ultra-cache\ignore\5.2.0\package.json'
✖ Error while installing levn@^0.4.1 from cache - Error reading package.json from C:\Users\Admin\.ultra-cache\levn\0.4.1\package.json - ENOENT: no such file or directory, open 'C:\Users\Admin\.ultra-cache\levn\0.4.1\package.json'
✖ Error extracting C:\Users\Admin\AppData\Local\Temp\ultra_tmp\lodash.merge-879fb9805f7a2dc7d3dd9b949be2071a.tgz to C:\Users\Admin\.ultra-cache\lodash.merge\4.6.2: TAR_BAD_ARCHIVE: Unrecognized archive format
✖ Error while installing eslint@8.26.0 from cache - Error reading package.json from C:\Users\Admin\.ultra-cache\lodash.merge\4.6.2\package.json - ENOENT: no such file or directory, open 'C:\Users\Admin\.ultra-cache\lodash.merge\4.6.2\package.json'
⚡ Installed packages in 3 min!
⚡ 1 Post Install scripts completed in 62.28 ms!
⚡ 121 packages verified
⚡ Done in 3 min!
```

### Expected Behavior

It should install those packages without any errors

### Template command (if applicable)

👇

### To Reproduce

1. Run `ultra create t3-app`
2. Say no to install packages
3. Install the packages manually with `ultra i`

**Comment by max-programming** [comment]
Although the app runs successfully, these errors seem like something is wrong

**Comment by nachoaldamav** [comment]
I'm aware of this problem, it happens when the download of a package had a problem, like slow internet or that an installation was cancelled.

**Comment by max-programming** [comment]
Yeah I have a slow internet connection 😅

**Comment by nachoaldamav** [comment]
I think I found a solution for this problem, it will be shipped with #132

**Comment by max-programming** [comment]
Cool! I wonder how you work with this system stuff. Never gotten into that

**Comment by nachoaldamav** [comment]
Should be fixed in v0.7.7!

`npm i ultra-pkg@latest -g`

**Comment by max-programming** [comment]
Cool! Will try it and let you know!
